tox.ini

```diff
@@ -41,7 +39,6 @@ deps = -rlib/galaxy/dependencies/pipfiles/flake8/pinned-hashed-requirements.txt
 [testenv:py27-lint-imports]
 commands = bash .ci/flake8_wrapper.sh
 whitelist_externals = bash
-skip_install = True
 deps = -rlib/galaxy/dependencies/pipfiles/flake8_imports/pinned-hashed-requirements.txt
 
 [testenv:py27-lint-imports-include-list]
```

Comment:
You can delete a `skip_install` also in this env.

Reply:
Done - thanks for the catch!